### DISCUSSION TITLE
When will try_recv be added back?

### DISCUSSION THREAD

**Discussion by UE2020** [tokio-rs/tokio#3896] [Q&A] [answered]
https://github.com/tokio-rs/tokio/pull/3263#issue-537783998
> The mpsc try_recv() functions have an issue where a sent message
happens-before a call to try_recv() but try_recv() returns None.
Fixing this is non-trivial, so the function is removed for 1.0. When the
bug is fixed, the function can be added back.

When will try_recv be added back?

**Comment by Darksonn** [comment, marked as answer]
When I find the time to properly review #3639. It's not been my top priority, and is relatively complex.
